ci/script.sh

```diff
@@ -20,7 +20,7 @@ case $TARGET in
 		sudo apt-get -y update
 		sudo apt-get install -y cmake pkg-config libssl-dev
 
-		cargo test --all --locked
+		cargo test --all --locked && cargo test --all --locked -- --ignored
```

Comment:
Running these tests in `--release` makes them a bit faster - maybe something to consider for the CI as well?

Reply:
For CI I think it's ok to be a bit slow. However the speed is more important for local development.

Reply:
there's no point running debug in CI and sometimes CI can be time critical so i'd prefer `--release` here